release.sh

```diff
@@ -41,17 +55,36 @@ if [ -n "$(git status --porcelain)" ]; then
     git commit --signoff -m "Update version to $VERSION"
 fi
 
-echo Linting, building and testing:
+echo "Linting, building and testing:"
 npm run lint
 npm run build
 npm run test
 
-echo Creating Tag...
+echo "Creating Tag ..."
 git tag -a -m "$VERSION_PREFIXED" "$VERSION_PREFIXED"
 
-echo Pushing version and tag to GitHub repository:
-git push
-git push "$(git config --get remote.origin.url)" "$VERSION_PREFIXED"
+if [[ $DRY_RUN ]]; then
+  echo "Pushing version and tag to GitHub repository (dry-run):"
+  git push --set-upstream origin "$RELEASE_BRANCH" --dry-run
+  git push "$(git config --get remote.origin.url)" "$VERSION_PREFIXED" --dry-run
+
+  echo "Publish to npmjs (dry-run):"
+  npm publish --access public --registry https://registry.npmjs.org/ --dry-run
+
+  echo "Deleting temporary release-branch (dry-run):"
+  git checkout main
+  git branch -D "$RELEASE_BRANCH"
+
+else
+  echo "Pushing version and tag to GitHub repository:"
+  git push --set-upstream origin "$RELEASE_BRANCH"
+  git push "$(git config --get remote.origin.url)" "$VERSION_PREFIXED"
+
+  echo "Publish to npmjs:"
+  npm publish --access public --registry https://registry.npmjs.org/
 
-echo Publish to npmjs:
-npm publish --access public --registry https://registry.npmjs.org/
+  echo "Deleting temporary release-branch:"
+  git checkout main
+  git push origin --delete "$RELEASE_BRANCH"
```

Comment:
We are using `origin` as the remote name for the central repository. But e.g. for me this would not work (`origin` points to my own fork of momo while `upstream` points to `TNG/momo-scheduler`)
We could try coming up with something nicer here (can we identify the remote based on the URL?), or need to document that the remote `origin` needs to point to `TNG/momo-scheduler`.

Reply:
Could we just used the url in the command directly?